how to migrate a project from umbraco 10 to 11 on cloud

jonat123 opened the pull request:
Created an article for how you can migrate a project from Umbraco 10 to 11

sofietoft commented on umbraco-cloud/upgrades/migrate-from-umbraco-10-to-11.md:
```diff
@@ -0,0 +1,161 @@
+---
+description:
+  In this article we show how you can upgrade your Umbraco 10 Cloud project locally to Umbraco 11 and then migrate the project to a new Umbraco 11 project.
```
```suggestion
description: "In this article we show how you can upgrade your Umbraco 10 Cloud project locally to Umbraco 11 and then migrate the project to a new Umbraco 11 project."
```

sofietoft commented on umbraco-cloud/upgrades/migrate-from-umbraco-10-to-11.md:
```diff
@@ -0,0 +1,161 @@
+  In this article we show how you can upgrade your Umbraco 10 Cloud project locally to Umbraco 11 and then migrate the project to a new Umbraco 11 project.
+---
+
+# Migrating a Umbraco 10 project to a Umbraco 11 project
```
Be aware: GitBook will automatically change the title here, based on what you called it in the SUMMARY file.

jonat123 replied:
Got it  👍

sofietoft commented on umbraco-cloud/upgrades/migrate-from-umbraco-10-to-11.md:
```diff
@@ -0,0 +1,161 @@
+
+# Migrating a Umbraco 10 project to a Umbraco 11 project
+
+Due to a temporary bug in the hosting services, we do not recommend to directly upgrade to Umbraco 11.
```
I don't think we need to mention this here. Wouldn't it make sense to instead keep the focus on the migration?

jonat123 replied:
Yeah makes sense, i am removing it!

sofietoft commented on umbraco-cloud/upgrades/migrate-from-umbraco-10-to-11.md:
```diff
@@ -0,0 +1,161 @@
+
+Due to a temporary bug in the hosting services, we do not recommend to directly upgrade to Umbraco 11.
+
+In this guide, we show how you can migrate your Umbraco 10 project to a new Umbraco 11 project on Umbraco Cloud instead.
```
```suggestion
In this guide, we will migrate an Umbraco Cloud project from Umbraco 10 to Umbraco 11.
```

sofietoft commented on umbraco-cloud/upgrades/migrate-from-umbraco-10-to-11.md:
```diff
@@ -0,0 +1,161 @@
+* Follow the **requirements** for [local development](../../umbraco-cms/fundamentals/setup/requirements.md#local-development).
+* A Umbraco Cloud project running **the latest Umbraco 10**
+* The .NET 7 Software Development Kit (SDK) installed locally.
+* A clean Umbraco 11 project.
```
Is this Cloud or non-Cloud? 🤔

jonat123 replied:
Its cloud :)

sofietoft commented on umbraco-cloud/upgrades/migrate-from-umbraco-10-to-11.md:
```diff
@@ -0,0 +1,161 @@
+
+<figure><img src="images/Solution-Explorer.png" alt=""><figcaption></figcaption></figure>
+
+* Select the **.Net 7** **Target Framework** drop-down in the **General** section of the **Application** tab as on your Cloud project.
```
This is quite confusing 🙈 
Perhaps try to split this up to simplify the steps a bit?

jonat123 replied:
I've tried to simplify it, let me know what you think!

sofietoft commented on umbraco-cloud/upgrades/migrate-from-umbraco-10-to-11.md:
```diff
@@ -0,0 +1,161 @@
+
+## Migrate Umbraco 10 project to Umbraco 11 project
+
+The following steps will guide you through the process of deploying your locally upgraded Umbraco 10 project to the **clean Umbraco 11 project**.
```
This is very similar to line 97 - consider only saying this once!

jonat123 replied:
Fixed it :)

sofietoft commented on umbraco-cloud/upgrades/migrate-from-umbraco-10-to-11.md:
```diff
@@ -0,0 +1,161 @@
+
+
+1. Clone down the **clean Umbraco 11 project**.
+2. Replace the `src/UmbracoProject/umbraco-cloud.json` file in the **old project** with the one from the **clean Umbraco 11 project**.
```
How about instead of calling it the "old project", we call it "the migration project" or something? 🤔 

Or, I remember doing this in similar guides previously, state somewhere in the guide what you will be calling the two projects going forward.
That way there's no confusion regarding which project is being referenced 🤔

jonat123 replied:
Makes sense to call it something else.

I think it could make sense to call it the same as its called in the prerequisites, just like I've done with the clean Umbraco 11 project 🤔

jonat123 replied:
Yeah makes sense to me :)

sofietoft commented on umbraco-cloud/upgrades/migrate-from-umbraco-10-to-11.md:
````diff
@@ -0,0 +1,161 @@
+    ```
+    git push origin master -f
+    ```
+4. Verify that the schema and files have been merged into the live environment on the **clean Umbraco 11 project**.
````
```suggestion
4. Verify that the schema and files have been merged into the Live environment on the **clean Umbraco 11 project**.
```

sofietoft reviewed (changes requested):
In general a good clean tutorial!

I've added some comments throughout - let me know if something doesn't make sense.

An additional comment: Is there any reason why you've chosen to use only unordered lists for most of the steps in this tutorial?

jonat123 commented on umbraco-cloud/upgrades/migrate-from-umbraco-10-to-11.md:
```diff
@@ -0,0 +1,161 @@
+* Umbraco.Cloud.Identity.Cms
+* Umbraco.Cloud.StorageProviders.AzureBlob
+
+{% hint style="info" %}
+Choose the package version corresponding to the CMS version that you are currently upgrading to.
+
+For example, if you are upgrading to "Umbraco.Cms 11.0.0" update the forms package to "Umbraco.Forms 11.0.0" as well.
+
+Also, if you have more projects in your solution or other packages, make sure that these are also updated to support tUmbraco 11.
+{% endhint %}
```
Hmm no you are right, i'll remove it

jonat123 commented:
@sofietoft Thanks for the review!

I've added your suggestions and changed some of the things based on your comments :) 

Let me know what you think!

sofietoft reviewed (approved):
LGTM 💪

jonat123 commented:
merging it in.
